string name
string description
string task_type
int8[] capabilities_needed
Pose pose
bool is_complete
float32 work_needed
float32 work_done
